src/gui/dialogs/unit_recruit.hpp

```diff
@@ -27,11 +27,13 @@ namespace dialogs
 class unit_recruit : public modal_dialog
 {
 public:
-	unit_recruit(std::vector<const unit_type*>& recruit_list, team& team);
+	/// @param recruit_list maps unit typs to strings. The strings are ""
```

Comment:
> There's no param called `recruit_list`.

Fixed

> Should it map to t_string instead of string?

What difference would that make?

Thanks for the review!

Reply:
I feel t_string makes it obvious that it's a string for showing to the user, because it's explicitly translatable.

Reply:
Done. Anything else before merging?